tests/integration/states/test_resource_group.py

```diff
@@ -1,7 +1,8 @@
 import pytest
 
 
-@pytest.mark.run(order=1)
+# @pytest.mark.run(order=1)
+@pytest.mark.skip(reason="resource group already exists")
```

Comment:
**chore (blocking):** if we're skipping a test in _all_ scenarios, we should remove it.